src/renderer/containers/ExplorePage.module.scss

```diff
@@ -13,6 +13,9 @@
   .card {
     border-radius: var.$radius-card;
     box-shadow: var.$shadow-card;
+    display: flex;
+    flex-direction: column;
+    gap: 10px;
```

Comment:
Are these changes relevant to this PR? :wink: In the future I'd be happy to see such changes in a separate PR. It can be tiny - but it makes my life as a reviewer more pleasant :grin:

Reply:
It's relevant to this - it's a small adjustment to add gaps between title, inputs and button in explore page